.github/workflows/compat_openssl3.yml

```diff
@@ -0,0 +1,43 @@
+name: Build for older MacOS (OpenSSL 3)
```

Comment:
We don't need this extra build.  The MacOS compat issues are orthogonal to OpenSSL 3.